docs/framework/configure-apps/how-to-enable-and-disable-automatic-binding-redirection.md

```diff
@@ -10,72 +10,81 @@ ms.author: "markl"
-  
-     ![Build warning for assembly reference conflicts](../../../docs/framework/configure-apps/media/clr-assemblyrefwarning.png "CLR_AssemblyRefWarning")  
-  
-2.  If there are assembly binding conflicts, a warning appears. Double-click the warning. (Keyboard: Select the warning and press **Enter**.)  
-  
-     A dialog box that enables you to automatically add the necessary binding redirects to the source web.config file appears.  
-  
-     ![Binding redirect permission dialog](../../../docs/framework/configure-apps/media/clr-addbindingredirect.png "CLR_AddBindingRedirect")  
-  
-## See Also  
- [\<bindingRedirect> Element](../../../docs/framework/configure-apps/file-schema/runtime/bindingredirect-element.md)  
- [Redirecting Assembly Versions](../../../docs/framework/configure-apps/redirect-assembly-versions.md)
+
+When you compile apps in Visual Studio that target the [!INCLUDE[net_v451](../../../includes/net-v451-md.md)] and later versions, binding redirects may be automatically added to the app configuration file to override assembly unification. Binding redirects are added if your app or its components reference more than one version of the same assembly, even if you manually specify binding redirects in the configuration file for your app. The automatic binding redirection feature affects traditional desktop apps and web apps that target the [!INCLUDE[net_v451](../../../includes/net-v451-md.md)] or a later version, although the behavior is slightly different for a web app. You can enable automatic binding redirection if you have existing apps that target previous versions of the .NET Framework, or you can disable this feature if you want to keep manually authored binding redirects.
+
+## Disable automatic binding redirects in desktop apps
+
+Automatic binding redirects are enabled by default for traditional desktop apps that target the [!INCLUDE[net_v451](../../../includes/net-v451-md.md)] and later versions. The binding redirects are added to the output configuration (app.config) file when the app is compiled and overrides the assembly unification that might otherwise take place. The source app.config file is not modified. You can disable this feature by modifying the project file for the app.
+
+1. Open the project file for editing using one of the following methods:
+
+   - In Visual Studio, select the project in **Solution Explorer**, and then choose **Open Folder in File Explorer** from the shortcut menu. In File Explorer, find the project (.csproj or .vbproj) file and open it in Notepad.
+   - In Visual Studio, in **Solution Explorer**, right-click the project and choose **Unload Project**. Right-click the unloaded project again, and then choose **Edit [projectname.csproj]**.
+
+2. In the project file, find the following property entry:
+
+   ```xml
+   <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+   ```
+
+3. Change **true** to **false**:
```

Comment:
Because these are verbatim code changes, should they be `code-fenced` instead of **bold**?

Reply:
I changed these to code-fenced.